### DISCUSSION TITLE
Tammikuun 2023 huoltoikkuna

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#361] [Tiedotteet]
Keskiviikon huoltokatkossa ei pystytä tekemään kaikkia suunniteltuja huoltoja, koska tarvittavat komponentit eivät ole saapuneet. Tämä tarkoittaa, että huolto tehdään loppuun helmikuun huoltokatkon aikana. 

Tästä johtuen luvassa on hieman oletettua vähemmän katkoilua huomisessa huoltoikkunassa, mutta aamulla on joka tapauksessa kaikkia kimppoja koskeva katko kun tietokantapalvelimen muistikammat vaihdetaan.

**Comment by CodoDerDritte** [comment]
Tietokantapalvelin 1 menee alas hetken kuluttua ja katko alkaa.

**Reply by CodoDerDritte** [reply]
Tietokantapalvelin 1 on nyt taas käytössä, joten Koha toimii jälleen ja kaikkia koskeva katko on ohi. Raporttien toimintaan toimintaan tulee vielä katko myöhemmin. Ilmoitan siitä erikseen.

Seuraavaksi alas menee sovellusnode 4, eli Amy. Tällä ei tässä kohtaa ole näkyvää vaikutusta mihinkään. Vaskissa ja Lapissa tulee Koha-kontainerin siirtymisestä aiheutuva muutaman minuutin katko kun Amyn muistikammat on saatu vaihdettua ja se saadaan takaisin käyntiin.

**Comment by CodoDerDritte** [comment]
Amyn muistit on vaihdettu. Siirrän Vaskin ja Lapin takaisin ajoon sinne, joten Vaskissa ja Lapissa tulee pieni katko.

**Comment by CodoDerDritte** [comment]
Vaski on takaisin kotinodellaan, eli katko Vaskin osalta ohi (lukuunottamatta niitä raportteja, joiden katko alkaa kohta).

**Comment by CodoDerDritte** [comment]
Lappi nyt myös palannut takaisin kotinodelleen, eli Lapin katkot ohi. Raporttien katko alkaa pian.

**Comment by CodoDerDritte** [comment]
Node 1:n kammat on nyt myös vaihdettu, joten siirrän Siilin, Lumpeen ja TäTin takaisin kotinodelleen. Näissä siis pienet muutaman minuutin katkot.

**Comment by CodoDerDritte** [comment]
Siili, Lumme ja TäTi takaisin ykkösellä, eli niiden katko ohi muuten kuin raporttien osalta. Raporttien katko käynnissä.

**Comment by CodoDerDritte** [comment]
Tietokanta 2 muistit on nyt myös vaihdettuna ja se on takaisin käynnissä, eli käyttökatkot ovat ohi. Nostelemme vielä testit kunhan ehdimme.

**Comment by CodoDerDritte** [comment]
SIP-serverit eivät näköjään halunneet käynnistyä automaattisesti konttien siirroissa, joten käynnistin ne käsin. Laitan asiakasviestinnän seuraavaksi takaisin käyntiin.

**Comment by CodoDerDritte** [comment]
Asiakasviestintä on taas käytössä ja testit nousevat pikkuhiljaa.

Homma on nyt valmis, eli katkot ovat tämän huoltoikkunan osalta ohi.
